### PR TITLE
#RET 1635 Create visibility filter file

### PR DESCRIPTION
**Pull request by ivonx**

### CODE REVIEW

File: src/main/resources/filter/web/visibility.filter [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+attribute=object-fit
+attribute=opacity
+attribute=order
+attribute=outline-color
+attribute=outline-style
+attribute=page-break-after
+attribute=page-break-before
+attribute=page-break-inside
+attribute=perspective
+  #  - perspective-origin
```
Comment (beatngu13) [inline src/main/resources/filter/web/visibility.filter]: Shouldn't this be:

```
#attribute=perspective-origin
```

Copy & paste from `attributes.yaml`? 😉

**Review by beatngu13 — DISMISSED** [review]
Same as in https://github.com/retest/recheck/pull/172.

File: src/main/resources/filter/web/visibility.filter [hunk 2]
```diff
@@ -0,0 +1,188 @@
+# Visibility filter file for recheck.
+
+# Ignore visibility CSS attributes:
+
+attribute=align-content
+attribute=align-items
+attribute=align-self
+attribute=all
```
Comment (roesslerj) [inline src/main/resources/filter/web/visibility.filter]: I think `all` is a retest internal attribute. I don't think that is an existing CSS attribute?

File: src/main/resources/filter/web/visibility.filter [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+attribute=cols
+attribute=colspan
+attribute=content
+attribute=high
+attribute=href
+attribute=hreflang
+attribute=kind
+attribute=label
+attribute=type
+attribute=value
```
Comment (roesslerj) [inline src/main/resources/filter/web/visibility.filter]: I would add `src` and newline at end of file.

**Review by roesslerj — CHANGES_REQUESTED** [review]

**Review by roesslerj — APPROVED** [review]

**Review by roesslerj — APPROVED** [review]

**Pull request merged**